src/Http/Validation/Domain.php

```diff
@@ -53,4 +55,32 @@ protected function stripProtocol($domain)
 
         return $domain;
     }
+
+    /**
+     * Strip the port from a domain.
+     *
+     * @param $domain
+     *
+     * @return mixed
+     */
+    protected function stripPort($domain)
+    {
+        $domainStripped = preg_replace(self::PATTERN_STRIP_PROTOCOL, '', $domain);
```

Comment:
This could only occur on malformed domain, strange we don't already have a failing test for that